tidb-cloud/tidb-node-group-management.md

```diff
@@ -0,0 +1,161 @@
+
+By default, you can create up to five TiDB node groups for a TiDB Cloud Dedicated cluster. If you need more groups, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). 
+
+If you create a TiDB node group but still use the endpoint of the default group to connect to the cluster, the TiDB nodes in the TiDB node group will not take any workload, which is a waste of the resource. You need to create new connection to the TiDB nodes in the new TiDB node group. See [Connect to a TiDB node group](#connect-to-a-tidb-node-group).
+
+## Connect to a TiDB node group
+
+### Connect via public connection
+
+Public connection for the new TiDB node group is disabled by default. You need to enable it first.
+
+To enable public connection, do the following:
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+2. In the upper-right corner, click **Connect**. A connection dialog is displayed.
+3. Choose the new TiDB node group and the **Public** connection type.
+4. Click **Configure the IP Access** List.
+5. On the **Networking** page, select the TiDB node group in the upper-right corner.
+6. Enable the public endpoint and add the IP access list.
+7. In the up-right corner of the page, click **Connect**, and you can get the connection string. 
+
+![Connect to the new TiDB node group via Public Endpoint](/media/tidb-cloud/tidb-node-group-connect-public-endpoint.png)
+
+For more information, see [Connect to TiDB Cloud Dedicated via Public Connection](/tidb-cloud/connect-via-standard-connection.md).
+
+### Connect via private endpoint
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+2. In the upper-right corner, click **Connect**. A connection dialog is displayed.
+3. Choose the new TiDB node group and **Private Endpoint** connection type.
+4. In the left navigation pane, click **Networking**. On the **Networking** page, select the TiDB node group.
+5. Click **Create Private Endpoint Connection** to create a new connection for this node group. For more information, see [Connect to a TiDB Cloud Dedicated Cluster via Private Endpoint with AWS](/tidb-cloud/set-up-private-endpoint-connections.md).
+
+    > **Note**:
+    >
+    > If you use Private Link to connect different node groups, you need to create separated private endpoint connection for each node group. 
+
+6. After you create the private endpoint connection, click **Connect** on the up-right of the page to get the connection string. 
+
+### Connect via VPC peering
+
+Because the cluster is in one VPC and all the TiDB node groups share the same VPC, you only need to create one VPC peering, then all the groups can use it. 
```

Comment:
```suggestion
Because all TiDB node groups share the same VPC as the cluster, you only need to create one VPC peering connection to enable access for all groups.
```